src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java

```diff
@@ -194,7 +194,7 @@ private static boolean structureBlockEqualsWorldBlock(
      * The internal structure loaded.
      */
     @Nullable
-    private final StructureWrapper theStructure;
+    private final com.ldtteam.structures.helpers.Structure theStructure;
```

Comment:
Probably import this

Reply:
It had the same name as this file that's why it didn't work, I renamed it to distinguish better